src/main/java/com/woowacourse/pelotonbackend/report/presentation/ReportController.java

```diff
@@ -0,0 +1,32 @@
+package com.woowacourse.pelotonbackend.report.presentation;
+
+import java.net.URI;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.woowacourse.pelotonbackend.report.application.ReportService;
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/api/reports")
```

Comment:
```suggestion
@RequestMapping("/api/reports")
@RestController
```
Request가 알파벳 순서 더 먼저에요 😄